internal/backends/mcu/stm32h735g.rs

```diff
@@ -0,0 +1,295 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+extern crate alloc;
+
+pub use cortex_m_rt::entry;
+use embedded_display_controller::{DisplayController, DisplayControllerLayer};
+use embedded_graphics::prelude::RgbColor;
+use hal::delay::Delay;
+use hal::gpio::Speed::High;
+use hal::ltdc::LtdcLayer1;
+use hal::pac;
+use hal::prelude::*;
+use hal::rcc::rec::OctospiClkSelGetter;
+use stm32h7xx_hal as hal;
+
+use defmt_rtt as _; // global logger
+
+#[cfg(feature = "panic-probe")]
+use panic_probe as _;
+
+#[alloc_error_handler]
+fn oom(layout: core::alloc::Layout) -> ! {
+    panic!("Out of memory {:?}", layout);
+}
+use alloc_cortex_m::CortexMHeap;
+
+use crate::{Devices, PhysicalRect, PhysicalSize};
+
+const HEAP_SIZE: usize = 200 * 1024;
```

Comment:
I guess the stack is shared with the heap in the regular RAM? Then the heap could also move to the 16MB space eventually.

Reply:
yes, we'll need to play with the memory.x